net/frr/src/opnsense/mvc/app/models/OPNsense/Quagga/RIP.xml

```diff
@@ -1,7 +1,7 @@
 <model>
     <mount>//OPNsense/quagga/rip</mount>
     <description>RIP Routing configuration</description>
-    <version>1.0.1</version>
+    <version>1.0.2</version>
```

Comment:
In theory not needed but it is better when it is there

Reply:
I know, only when default is yes, but I wasn't sure regarding those migration errors during upgrade.